docs/networking/guide/ipv6-ip-planning.md

```diff
@@ -3,7 +3,7 @@ title: Conceptual planning for IPv6 networking
 description: Learn strategies for transitioning an IPv4 network environment on Azure to IPv6. IPv6 provides a larger pool of internet addresses to accommodate growth.
 author: bsteph
 ms.author: bstephenson
-ms.date: 01/08/2024
+ms.date: 06/25/2024
```

Comment:
Did a complete end-to-end review of this article get complete?  If not, commit this suggestion.

```suggestion
ms.date: 01/08/2024
```